tests/dummy/app/pods/docs/cookbook/addon-configs/template.md

```diff
@@ -0,0 +1,18 @@
+
+# Addon Configs
+
+## Filter Locales
+
+If you need to distribute applications separately in different languages, now we can use `includeLocales` or `excludeLocales` configuration options.
+
+```
+// config/ember-intl.js
+{
+  includeLocales: ['en-us', 'zh-cn'],
```

Comment:
could/should this be case insensitive?

Reply:
em, you are right . `normalize-locale`

Reply:
hi,@jasonmit , i see the config in broccoli-funnel, include /exclude glob string no support case-insensitive (not use minimatch `nocase` )
so , this case , can we use it like below ？
```
includeLocales: ['en-*', 'zh-*'] 
```

or i need refactor this pr , not a new filter tree , but modify translation-reducer .